tests/no_segfault_on_thread_exit.rs

```diff
@@ -0,0 +1,36 @@
+use std::iter::repeat_with;
+use std::thread;
+
+#[test]
+fn no_segfault_on_thread_exit() {
```

Comment:
For my understanding: `--nowin64_unwinding_info` disables the buggy code path? I'll commit this to try it out, feel free to remove again.
```suggestion
fn no_segfault_on_thread_exit() {
  v8::V8::set_flags_from_string("--nowin64_unwinding_info");
```